WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsPresenting.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+protocol CardPresentPaymentsPresenting {
```

Comment:
❓  This doesn't seem to be used in this PR so far, and looks identical to `ViewControllerPresenting` in https://github.com/woocommerce/woocommerce-ios/pull/12896. How do you plan to use this protocol?

Reply:
Oops. I just left it in having planned to use the generic one. Thanks!